### PR TITLE
Added iOS 7 note

### PR DESCRIPTION
**Pull request by chri2547**

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-bbf367d662/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-ec1f885530/how-to/)

File: content/exchange/manually-configure-ios-devices-for-email-hosted-on-exchange-2013.md [hunk 1]
```diff
@@ -5,36 +5,40 @@ title: Manually configure iOS devices for email hosted on Exchange 2013
...
 created_by: Mawutor Amesawu
-last_modified_date: '2016-01-28'
-last_modified_by: Rose Coste
+last_modified_date: '2019-03-28'
+last_modified_by: Chris Moyer
 product: Microsoft Exchange
 product_url: exchange
 ---
 
 This article describes how to configure an Exchange 2013 account on your
```
Comment (cyrichardson) [inline content/exchange/manually-configure-ios-devices-for-email-hosted-on-exchange-2013.md]: Microsoft&reg; Exchange&reg;

File: content/exchange/manually-configure-ios-devices-for-email-hosted-on-exchange-2013.md [hunk 2]
```diff
@@ -5,36 +5,40 @@ title: Manually configure iOS devices for email hosted on Exchange 2013
...
-Apple iOS device.  The administrator of your account must verify that an
+Apple iOS 7 device.
```
Comment (cyrichardson) [inline content/exchange/manually-configure-ios-devices-for-email-hosted-on-exchange-2013.md]: Apple&reg;

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-df3541e432/how-to/)

File: content/exchange/manually-configure-ios-devices-for-email-hosted-on-exchange-2013.md [hunk 3]
```diff
@@ -5,36 +5,40 @@ title: Manually configure iOS devices for email hosted on Exchange 2013
...
+
+**Note:** The images in this article show iPhone 5 running iOS 7. Although
```
Comment (cyrichardson) [inline content/exchange/manually-configure-ios-devices-for-email-hosted-on-exchange-2013.md]: iPhone&reg;

File: content/exchange/manually-configure-ios-devices-for-email-hosted-on-exchange-2013.md [hunk 4]
```diff
@@ -5,36 +5,40 @@ title: Manually configure iOS devices for email hosted on Exchange 2013
...
+screens on other iOS devices differ, the process is similar. For more modern iOS versions, refer to the [Cloud Office Email Help Tool](https://emailhelp.rackspace.com) for instructions.
+
+The administrator of your account must verify that an
 ActiveSync license is attached in your account via the Cloud Control
```
Comment (cyrichardson) [inline content/exchange/manually-configure-ios-devices-for-email-hosted-on-exchange-2013.md]: ActiveSync&reg;

File: content/exchange/manually-configure-ios-devices-for-email-hosted-on-exchange-2013.md [hunk 5]
```diff
@@ -5,36 +5,40 @@ title: Manually configure iOS devices for email hosted on Exchange 2013
...
-Panel.  If any Internet connectivity issues prevent you from
+Panel. If any Internet connectivity issues prevent you from
 successfully setting up Exchange, contact your wireless provider or
 ISP.
```
Comment (cyrichardson) [inline content/exchange/manually-configure-ios-devices-for-email-hosted-on-exchange-2013.md]: Internet Service Provider (ISP)

Comment (chri2547) [reply content/exchange/manually-configure-ios-devices-for-email-hosted-on-exchange-2013.md]: Done.

File: content/exchange/manually-configure-ios-devices-for-email-hosted-on-exchange-2013.md (hunk 1 above)
Comment (chri2547) [reply content/exchange/manually-configure-ios-devices-for-email-hosted-on-exchange-2013.md]: Won't do as we don't include a registered trademark symbol in metadata.

Comment (chri2547) [reply content/exchange/manually-configure-ios-devices-for-email-hosted-on-exchange-2013.md]: Done.

File: content/exchange/manually-configure-ios-devices-for-email-hosted-on-exchange-2013.md (hunk 2 above)
Comment (chri2547) [reply content/exchange/manually-configure-ios-devices-for-email-hosted-on-exchange-2013.md]: Done.

File: content/exchange/manually-configure-ios-devices-for-email-hosted-on-exchange-2013.md (hunk 3 above)
Comment (chri2547) [reply content/exchange/manually-configure-ios-devices-for-email-hosted-on-exchange-2013.md]: Done.

File: content/exchange/manually-configure-ios-devices-for-email-hosted-on-exchange-2013.md (hunk 4 above)
Comment (chri2547) [reply content/exchange/manually-configure-ios-devices-for-email-hosted-on-exchange-2013.md]: Done.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-3164e2b3a1/how-to/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-e491e610fa/how-to/).

**Pull request merged**